AIOnboardingARMTemplates/existingClusterOnboarding.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aksResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "AKS Cluster Resource ID"
+            }
+        },
+        "aksResourceLocation": {
+            "type": "string",
+            "metadata": {
+              "description": "Location of the AKS resource e.g. \"East US\""
+            }
+        },
+        "openTelemetryLogsPort": {
+            "type": "int",
+            "metadata": {
+              "description": "Port to use for OpenTelemetry Logs"
+            },
+            "defaultValue": "28331"
+        },
+        "openTelemetryMetricsPort": {
+            "type": "int",
+            "metadata": {
+              "description": "Port to use for OpenTelemetry Metrics"
+            },
+            "defaultValue": "28333"
+        },
+        "resourceTagValues": {
+            "type": "object",
+            "metadata": {
+              "description": "Existing or new tags to use on AKS, ContainerInsights and DataCollectionRule Resources"
+            }
+        }
+    },
+    "variables": {
+        "clusterSubscriptionId": "[split(parameters('aksResourceId'),'/')[2]]",
+        "clusterResourceGroup": "[split(parameters('aksResourceId'),'/')[4]]",
+        "clusterName": "[split(parameters('aksResourceId'),'/')[8]]",
+        "clusterLocation": "[replace(parameters('aksResourceLocation'),' ', '')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
+            "name": "[Concat('aks-appmonitoring-msi-addon', '-',  uniqueString(parameters('aksResourceId')))]",
+            "apiVersion": "2017-05-10",
+            "subscriptionId": "[variables('clusterSubscriptionId')]",
+            "resourceGroup": "[variables('clusterResourceGroup')]",
+            "properties": {
+              "mode": "Incremental",
+              "template": {
+                "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+                "contentVersion": "1.0.0.0",
+                "parameters": {},
+                "variables": {},
+                "resources": [
+                    {
+                        "name": "[variables('clusterName')]",
+                        "type": "Microsoft.ContainerService/managedClusters",
+                        "location": "[parameters('aksResourceLocation')]",
+                        "tags": "[parameters('resourceTagValues')]",
+                        "apiVersion": "2024-02-02-preview",
```

Comment:
Is this the new _api-version_ that has the latest changes?

Reply:
yes this is the new version